src/main/java/org/rutebanken/tiamat/exporter/PublicationDeliveryExporter.java

```diff
@@ -91,7 +91,7 @@ public PublicationDeliveryStructurePage exportStopPlacesWithEffectiveChangeInPer
     public PublicationDeliveryStructure createPublicationDelivery() {
         PublicationDeliveryStructure publicationDeliveryStructure = new PublicationDeliveryStructure()
                 .withVersion(ANY_VERSION)
-                .withPublicationTimestamp(OffsetDateTime.now())
+                .withPublicationTimestamp(LocalDateTime.now())
```

Comment:
This has been resolved in our master. Please update your fork from Entur master. One tip is to create a new branch from Entur master for a new PR. Then merge it to your fork's master, and create PR to us :+1: